adal/src/Microsoft.IdentityModel.Clients.ActiveDirectory/Platforms/android/PlatformParameters.cs

```diff
@@ -59,11 +59,24 @@ public PlatformParameters(Activity callerActivity, bool useBroker):this(callerAc
         /// <param name="callerActivity">activity calling ADAL to acquire token</param>
         /// <param name="useBroker">flag to enable or disable broker flow. FALSE by default.</param>
         /// <param name="promptBehavior">Prompt behavior enum to control UI</param>
-        public PlatformParameters(Activity callerActivity, bool useBroker, PromptBehavior promptBehavior)
+        public PlatformParameters(Activity callerActivity, bool useBroker, PromptBehavior promptBehavior) :base(callerActivity, useBroker, promptBehavior, 0, 1001)
```

Comment:
>base [](start = 107, length = 4)

this ?

Reply:
This was in error. It should have referenced the constructor overload instead of the base class. I have amended the request.